be/src/util/c_string.h

```diff
@@ -14,6 +14,8 @@
 
 #pragma once
 
+#include <stdint.h>
```

Comment:
already included in https://github.com/StarRocks/starrocks/pull/49351/files#diff-8263193bc080d7ba8dc61bf8414b552be86a6cb0c5ba8fc88c8dce147a343add

this line can be removed now.

Reply:
done